src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -176,8 +216,35 @@ export const ChannelView: React.FC = () => {
           </StyledButtonContainer>
         </TitleSection>
         <TabsContainer>
-          <Tabs initialIndex={0} tabs={mappedTabs} onSelectTab={handleSetCurrentTab} />
+          <StyledTabs
+            selected={isSearching ? -1 : undefined}
+            initialIndex={0}
+            tabs={mappedTabs}
+            onSelectTab={handleSetCurrentTab}
+          />
           {currentTabName === 'Videos' && (
+            <SearchContainer>
```

Comment:
As above - this could be a form element

Reply:
It didn't work at the end we did need the keypresses

Reply:
but why?
I've just applied to your code: 
```tsx
// SearchContainer needs to be form element
          <SearchContainer
              onSubmit={(e) => {
                e.preventDefault()
                if (searchQuery.trim() === '') {
                  setSearchQuery('')
                  setIsSearching(false)
                } else {
                  search()
                  setIsSearching(true)
                }
              }}
            >
              <StyledTextField
                ref={searchInputRef}
                isOpen={isSearchInputOpen}
                value={searchQuery}
                onBlur={() => {
                  setIsSearchingInputOpen(false)
                  setSearchQuery('')
                }}
                onChange={(e) => setSearchQuery(e.target.value)}
                // onKeyDown={handleKeyPress}
                placeholder="Search"
                type="search"
                helperText={null}
              />

```
and everything works just fine. I've added onBlur event too. Not sure why we need an event that is firing on the ESC key, but I guess in the end we could keep it. 

Also, I still think we should use apollo's `lazy query` here.

Reply:
My bad didn't notice that you used lazy query after all 😅

Reply:
Yep ESC stuff was by design on the figma prototype